config/locales/client.en.yml

```diff
@@ -4033,6 +4033,7 @@ en:
         go_back: "Back to Search"
         recommended: "We recommend customizing the following text to suit your needs:"
         show_overriden: "Only show overridden"
+        more_than_50_results: "There are more than 50 results. Please refine your search."
```

Comment:
I would suggest something like "Many results were found. The first 50 results are shown." to be more clear about what was shown.

Reply:
In the locales there is also the key `site_settings.more_than_30_results` which is `"There are more than 30 results. Please refine your search or select a category."`

Maybe keep it like this so the wording is consistent for this type of error?

Reply:
Sure sounds good. I would change it but it's better to change both so  let's leave as is.